js/components/controlBar.js

```diff
@@ -10,6 +10,7 @@ var React = require('react'),
 var ControlBar = React.createClass({
   getInitialState: function() {
     this.isMobile = this.props.controller.state.isMobile;
+    this.responsiveUIMultiple = Utils.responsiveUIMultiple(this.props.responsiveView);
     return {
```

Comment:
the multiple is available in the skin.json -

`this.props.skinConfig.responsive.breakpoints.sm.multiple`